prac_05/emails.py

```diff
@@ -20,6 +21,7 @@ def main():
 
 
 def get_name_from_email(email):
+    """Extract name from emails"""
     start = email.split("@")[0]
```

Comment:
Good splitting of the work here, clear to see whats happening at each line

Reply:
cheers